README.md

```diff
@@ -10,6 +10,15 @@ CZ CELLxGENE Explorer (pronounced "cell-by-gene") is an interactive data explore
 
 Whether you need to visualize one thousand cells or one million, CELLxGENE Explorer helps you gain insight into your single-cell data.
 
+#### Highlighted Features
+
+- **Interactive Exploration**: Provides intuitive navigation of 2D matrices with dimensions, allowing users to explore and annotate datasets.
```

Comment:
We currently doesn't have user annotations 😭 So dropping the word for now. Thank you!

```suggestion
- **Interactive Exploration**: Provides intuitive navigation of 2D matrices with dimensions, allowing users to explore datasets.
```